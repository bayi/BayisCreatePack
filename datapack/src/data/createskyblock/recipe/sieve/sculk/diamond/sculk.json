{
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "minecraft:sculk"
  },
  "mesh": {
    "item": "exdeorum:diamond_mesh"
  },
  "result": {
    "count": 1,
    "id": "minecraft:sculk"
  },
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.25
  }
}
