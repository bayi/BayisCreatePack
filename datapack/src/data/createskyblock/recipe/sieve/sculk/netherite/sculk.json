{
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "minecraft:sculk"
  },
  "mesh": {
    "item": "exdeorum:netherite_mesh"
  },
  "result": {
    "count": 1,
    "id": "minecraft:sculk"
  },
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.5
  }
}
