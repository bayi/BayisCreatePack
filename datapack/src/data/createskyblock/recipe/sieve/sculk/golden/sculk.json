{
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "minecraft:sculk"
  },
  "mesh": {
    "item": "exdeorum:golden_mesh"
  },
  "result": {
    "count": 1,
    "id": "minecraft:sculk"
  },
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.18
  }
}
